### PR TITLE
Integration CLI tests

### PR DESCRIPTION
**Pull request by AshwinHIBM**
Job to trigger scripts in https://github.com/ppc64le-cloud/docker-ce-build/pull/225
These tests will run against https://github.com/docker/cli

### CODE REVIEW

**Comment by AshwinHIBM** [comment]
@Rajalakshmi-Girish https://github.com/ppc64le-cloud/docker-ce-build/pull/225 merged. Can you PTAL at this PR?

File: config/jobs/periodic/docker/periodic-ci-docker.yaml [hunk 1]
```diff
@@ -123,6 +123,47 @@ periodics:
             exit $rc
           securityContext:
             privileged: true
+    - name: periodic-integration-cli-docker
+      cluster: k8s-ppc64le-cluster
+      labels:
+              preset-build-docker: "true"
```
Comment (Rajalakshmi-Girish) [inline config/jobs/periodic/docker/periodic-ci-docker.yaml:129]: The indentation space at line 129 can be reduced.

File: config/jobs/periodic/docker/periodic-ci-docker.yaml [hunk 2]
```diff
@@ -123,6 +123,47 @@ periodics:
...
+      cron: 0 14 * * *
```
Comment (Rajalakshmi-Girish) [inline config/jobs/periodic/docker/periodic-ci-docker.yaml:130]: The time set at cron clashes with the existing job `periodic-integration-test-docker`'s cron time.
Choose a time stamp at which no other job gets trigger.

File: config/jobs/periodic/docker/periodic-ci-docker.yaml [hunk 3]
```diff
@@ -123,6 +123,47 @@ periodics:
...
+          channel: 'prow-job-notifications'
+          report_template: 'Job {{.Spec.Job}} of type {{.Spec.Type}} ended with state {{.Status.State}}. <!subteam^S02N6DWBX0F> <{{.Status.URL}}|View logs>'
+      spec:
+        containers:
+        - image: quay.io/powercloud/docker-ce-build@sha256:f93b6d36ad414d30f584c6555d8c2116de58bd56601da36f4e13d4b5b624114a
+          resources:
+            requests:
+              cpu: "4000m"
+            limits:
+              cpu: "4000m"
```
Comment (Rajalakshmi-Girish) [inline config/jobs/periodic/docker/periodic-ci-docker.yaml:149]: Hope you have test run this job in your development cluster and visualized the view of test results in junit tab?

File: config/jobs/periodic/docker/periodic-ci-docker.yaml [hunk 4]
```diff
@@ -123,6 +123,47 @@ periodics:
...
+      reporter_config:
+        slack:
+          channel: 'prow-job-notifications'
+          report_template: 'Job {{.Spec.Job}} of type {{.Spec.Type}} ended with state {{.Status.State}}. <!subteam^S02N6DWBX0F> <{{.Status.URL}}|View logs>'
+      spec:
+        containers:
+        - image: quay.io/powercloud/docker-ce-build@sha256:f93b6d36ad414d30f584c6555d8c2116de58bd56601da36f4e13d4b5b624114a
+          resources:
+            requests:
+              cpu: "4000m"
```
Comment (Rajalakshmi-Girish) [inline config/jobs/periodic/docker/periodic-ci-docker.yaml:147]: Are you sure this job would also need 4000m cpu like other docker jobs? 
As this one aims at testing CLI, I suppose a lesser resource should suffice. Can you analyse on how much is the cpu required by this job?(Compare resource usage with similar job run upstream on x86)

File: config/jobs/periodic/docker/periodic-ci-docker.yaml (hunk 2 above)
Comment (AshwinHIBM) [reply config/jobs/periodic/docker/periodic-ci-docker.yaml]: Adjusted `periodic-integration-test-docker` to 14:30 https://github.com/ppc64le-cloud/test-infra/compare/master...AshwinHIBM:test-infra:integration-cli#diff-d529e4fce74de0ef2b5f5f6f4131d73c755e5895ad4749261c3a15a2bcc6e61dR171

File: config/jobs/periodic/docker/periodic-ci-docker.yaml (hunk 4 above)
Comment (AshwinHIBM) [reply config/jobs/periodic/docker/periodic-ci-docker.yaml]: The upstream job uses 4000m but my experiment showed I can safely run `periodic-integration-cli-docker` with 2000 and `periodic-integration-test-docker` with 4000.
https://github.com/ppc64le-cloud/test-infra/compare/master...AshwinHIBM:test-infra:integration-cli#diff-d529e4fce74de0ef2b5f5f6f4131d73c755e5895ad4749261c3a15a2bcc6e61dR147
https://github.com/ppc64le-cloud/test-infra/compare/master...AshwinHIBM:test-infra:integration-cli#diff-d529e4fce74de0ef2b5f5f6f4131d73c755e5895ad4749261c3a15a2bcc6e61dR147

**Comment by AshwinHIBM** [comment]
Commit https://github.com/ppc64le-cloud/test-infra/pull/483/commits/508c60a1f92f1a77fa9cd1b92f561072e37deb85 is for setting up the `integration-cli-docker` job and commit https://github.com/ppc64le-cloud/test-infra/pull/483/commits/f61554d143832455c9714832196ac8349ee87b91 is for adjusting the CPU usage and timing of the `periodic-integration-test-docker` job. Please note, the collective diff view of the PR isn't showing colours as I would expect.

**Pull request closed without merging**